### DISCUSSION TITLE
How to implement the "expand all" functionality for an accordion using react?

### DISCUSSION THREAD

**Discussion by jlara005** [department-of-veterans-affairs/vets-design-system-documentation#881] [Announcements]
Hi,

Does someone know how to implement the "expand all" functionality for an accordion using react? I don't see this in the examples.

https://design.va.gov/components/accordion

![image](https://user-images.githubusercontent.com/29823126/169070147-e3f30712-96ae-4401-8cb4-d2e6f9ae6bbf.png)

**Comment by bkjohnson** [comment]
That button & functionality is already included in the default accordion, no extra work is needed. You just use the accordion web component in your JSX or HTML like this:

```html
<va-accordion>
  <va-accordion-item>
    <h3 slot="headline">First Item</h3>
    Item body text
  </va-accordion-item>
  <va-accordion-item>
    <h3 slot="headline">Second Item</h3>
    Body text for different item
  </va-accordion-item>
</va-accordion>
```

One thing to be aware of is that this feature was added in [version `8.5.0` of the component-library](https://github.com/department-of-veterans-affairs/component-library/releases/tag/v.8.5.0), so you will only see it if you are using that version or a later one.

**Reply by jlara005** [reply]
We are using 6.2.1 so it's definitely version. Thanks @bkjohnson!
